locations/jup.yml

```diff
@@ -65,8 +65,8 @@ networks:
     name: mesh_bht
     prefix: 10.31.147.128/32
     ipv6_subprefix: -1
-    mesh_metric: 2048
-    mesh_metric_lqm: ['default 0.25']
+    mesh_metric: 1024
+    mesh_metric_lqm: ['default 0.5']
```

Comment:
why the olsr change?

Reply:
0.25 was introduced due to the outage of the bht-jup antenna. The outage is fixed and the link has almost the same bandwidth as the segen link now, but segen still has fewer hops. Therefore no need to make it that bad anymore.